HARSH/script.js

```diff
@@ -0,0 +1,83 @@
+const resultBox = document.querySelector("#result-box")
```

Comment:
The Js is fine:) Although there"s room for "displaying what we are calculating " on the resultBox